tests/data/car.py

```diff
@@ -138,11 +138,7 @@ def extract_production_date(snapshot):
 
 
 def build_all_default_config():
-    return {
-        "production_date": datetime.datetime(2000, 1, 1),
-        "owner": {},
-        "incidents": [],
-    }
+    return {"production_date": datetime.datetime(2000, 1, 1)}
```

Comment:
This is a revert of changes done in #123. I could have realised already here the issue this PR is addressing, but was to deep down into the interplay between allowing none, defaults and being required...